client/blocks/reader-post-card/style.scss

```diff
@@ -595,13 +595,19 @@
 	display: flex;
 	margin: 0 0 17px;
 	padding: 0;
+	position: relative;
 }
 
 .reader-post-card__gallery-item {
 	border: 1px solid var( --color-neutral-0 );
 	cursor: pointer;
 	flex: 1;
+	height: 300px;
```

Comment:
I haven't tested this but will the `reader-post-card__gallery-item` always be 598x300, even on mobile?

Reply:
Updated this in https://github.com/Automattic/wp-calypso/pull/67387/commits/2b98ad641eced13ce7b771c1b937ab92d48184f6